.github/workflow/prettier.yml

```diff
@@ -0,0 +1,25 @@
+name: Continuous Integration
+
+# This action works with pull requests and pushes
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  prettier:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          # Make sure the actual branch is checked out when running on pull requests
+          ref: ${{ github.head_ref }}
+
+      - name: Prettify code
+        uses: creyD/prettier_action@v4.2
+        with:
+          # This part is also where you can pass other options, for example:
+          prettier_options: --write **/*.{js,md}
```

Comment:
as said in private maybe add 
`commit_options`
or `push_commit`
or `same_commit`
see https://github.com/creyD/prettier_action

Reply:
I added `check` instead of write, to avoid modifying the code automatically, which can be a bit weird.
The action will block the PR if the code isn't formatted correctly